app/controllers/check_ins_controller.rb

```diff
@@ -1,7 +1,7 @@
 # /check_ins/{new, create}
 class CheckInsController < ApplicationController
   def new
-    @check_in = CheckIn.new
+    @check_in = CheckIn.new(check_in_params)
```

Comment:
Yep. This way when we show the check in form it is auto-filled with the params that are passed in via the URL